addons/project/models/project.py

```diff
@@ -854,7 +854,7 @@ def get_panel_data(self):
             panel_data['milestones'] = self._get_milestones()
         if self._show_profitability():
             profitability_items = self._get_profitability_items()
-            if self._get_profitability_sequence_per_invoice_type() and profitability_items and 'revenues' in profitability_items and 'costs' in profitability_items:  # sort the data values
+            if self._get_profitability_sequence_per_invoice_type() and profitability_items and 'revenues' in profitability_items and 'costs' in profitability_items and profitability_items['revenues']['data']:  # sort the data values
```

Comment:
If we only have an MO in draft, meaning no analytic account and no sequence in profitability_items['costs']['data'], there is no need to sort the data; otherwise, a traceback will occur.

Reply:
To be clear, I will not allow to add a category just to have a button to get MO linked to project. We should include the MO in draft in the compute of the project profitability.